TombEngine/Game/collision/collide_room.h

```diff
@@ -126,6 +126,7 @@ CollisionResult GetCollision(ItemInfo* item);
 CollisionResult GetCollision(ItemInfo* item, short headingAngle, float forward, float down = 0.0f, float right = 0.0f);
 CollisionResult GetCollision(Vector3i pos, int roomNumber, short headingAngle, float forward, float down = 0.0f, float right = 0.0f);
 CollisionResult GetCollision(int x, int y, int z, short roomNumber);
+CollisionResult GetCollision(GameVector point);
```

Comment:
Just an organisational thing, please make it `CollisionResult GetCollision(const GameVector& point);`